docs/markdown/Builtin-options.md

```diff
@@ -275,6 +275,21 @@ is inherited from the main project. This is useful, for example, when the main
 project requires C++11, but a subproject requires C++14. The `cpp_std` value
 from the subproject's `default_options` is now respected.
 
+Since *1.2.0* `c_std` and `cpp_std` options now accepts a list of values.
+Projects that prefer GNU C but can fallback to ISO C can now set for
+example `default_options: 'c_std=gnu11,c11'` and it will use gnu11 when
```

Comment:
> , but can fallback to ISO C, can now set, for example, `default_options: 'c_std=gnu11,c11'`, and it will use `gnu11` when available, but fallback to `c11` otherwise.